src/HttpRequest.cc

```diff
@@ -53,15 +53,15 @@ HttpRequest::HttpRequest(const HttpRequestMethod& aMethod, AnyP::ProtocolType aP
 {
     assert(mx);
     static unsigned int id = 1;
-    debugs(93,7, HERE << "constructed, this=" << this << " id=" << ++id);
+    debugs(93,7, "constructed, this=" << this << " id=" << ++id);
```

Comment:
This PR automatically removes HERE, and nothing else should be required. The spacing issue is covered in another comment, so I will not repeat those arguments here.